### PR TITLE
hotfix nix install URL: set to 2.18.5

### PR DESCRIPTION
**Pull request by roberth**

### CODE REVIEW

File: src/content/download/01-nix-linux.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ platform: linux
 Install Nix via the <span class="font-bold">recommended</span> [multi-user installation](/manual/nix/stable/installation/multi-user):
 
 ```bash
-$ sh <(curl -L https://nixos.org/nix/install) --daemon
+$ sh <(curl -L hhttps://releases.nixos.org/nix/nix-2.18.5/install) --daemon
```
Comment (RossComputerGuy) [inline src/content/download/01-nix-linux.mdx:12]: Is there supposed to be two h's in https?

**Comment by thilobillerbeck** [comment]
uhm, it looks alright, but is it really meant to be to be set to `2.18.5` again? Both links are working for me atm

**Comment by roberth** [comment]
We've figured it out the right way, and all the way to 2.18 was a bit excessive anyway.
Sorry for the noise.

**Comment by thilobillerbeck** [comment]
no problem :) glad that this is resolved. :+1:  Thanks for fixing this so quickly! :D

**Comment by cole-h** [comment]
https://github.com/NixOS/nixos-homepage/pull/1540 is related, that one _could_ use a merge pretty please :)

**Pull request closed without merging**